node/chainSimulator/components/networkComponents.go

```diff
@@ -27,7 +27,7 @@ type networkComponentsHolder struct {
 }
 
 // CreateNetworkComponentsHolder creates a new networkComponentsHolder instance
-func CreateNetworkComponentsHolder(network SyncedBroadcastNetworkHandler) (*networkComponentsHolder, error) {
+func CreateNetworkComponentsHolder(network SyncedBroadcastNetworkHandler) (factory.NetworkComponentsHandler, error) {
```

Comment:
Please rename to `CreateNetworkComponents` and remove the inner closeHandler declaration & usage

Reply:
done